Make OTP installation optional (#81)
erlef/setup-beam: README.md

This action sets up an Erlang/OTP environment for use in a GitHub Actions
workflow by:

> - installing Erlang/OTP

Reviewer: ❤️

      - run: rebar3 ct
```

> ### Example (Gleam + `rebar3`, on Ubuntu)

Reviewer: Do we want to get rid of the Gleam + `rebar3` example? I don't mind that much, but I don't understand why.

Author: Gleam switched to using its own binary as a build tool as the default, this likely will be the best supported way of building Gleam projects. And I thought it'd be weird having three examples specific to Gleam

Reviewer: Sure thing. That makes sense.

      - uses: erlef/setup-beam@v1
        with:
          otp-version: '24'
          gleam-version: '0.19.0-rc3'
      - run: gleam test
```

### Example (Gleam on Ubuntu without OTP)

```yaml
# create this in .github/workflows/ci.yml
on: push

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: erlef/setup-beam@v1
        with:
          otp-version: false
          gleam-version: '0.19.0-rc3'
      - run: gleam check
```

> **Note**: The `otp-version: false` config is only applicable when installing Gleam.

Reviewer: ```suggestion
**Note**: the `otp-version: false` input is only applicable when installing Gleam.
```

Reviewer: Regarding the small caps, I'll be fixing the rest of the README in a different pull request.

      - uses: erlef/setup-beam@v1
        with:
          otp-version: '24'
          gleam-version: '0.19.0-rc3'
      - run: gleam test
```

### Example (Gleam on Ubuntu without OTP)

> 

Reviewer 2: ```suggestion
Note that `otp: false` currently is only applicable to gleam. 
```

Reviewer 2: @paulo-ferraz-oliveira Thoughts?

Author: I included this note below the example snippet. Should it be moved up?

Reviewer 2: Ahh, nah that's good. I just didn't see, it should read fine in the actual README vs a PR :)

- `@v1.8`: the latest in the `1.8.z` series (this tag is movable),
- `@v1.8.0`: release `1.8.0` (this tag is not movable).

We make a real effort to not introduce incompatibilities without changing the major
version number. To be extra safe against changes causing issues in CI you can specify

> an exact version with `@vx.y.z`.

Author: I found this part a bit confusing so I took liberty to change it. I think this is more clear. Or maybe I misunderstood and specifying all three numbers is _always_ recommended?

Reviewer: It's not _always_ recommended, but it's preferable if you want a more stable environment (and even so, the underlying Node.js packages change, so it's really not just up to us, unfortunately). Your wording is better than what was there before, in any case.

> version number. To be extra safe against changes causing issues in CI you can specify
> an exact version with `@vx.y.z`.

Reviewer: ```suggestion
version number. To be extra safe against changes causing issues in your CI you should specify
an exact version with `@vx.y.z`.
```
